openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
```

Comment:
lol, fwiw, 'kill' is fairly standard nomenclature around thread/process termination, macabre as it may be.  'stop' is typically used for system-level services and implies start/stop/restart semantics.